src/material/chips/chip.scss

```diff
@@ -695,16 +695,21 @@ $_avatar-trailing-padding: 8px;
 
   // Used as a state overlay.
   &::after {
```

Comment:
This is better, but now the hover ripple is giant and off-center. I think you can undo the changes below and just add the following styles:
```
    box-sizing: border-box;
    padding: 12px; // should figure out how to calculate this, logically its the (touch-target-size - ripple-size)/2
    margin: -12px;
    background-clip: content-box;
```

Reply:
Hi @mmalerba, to make sure the touch-target meets the accessible 48px x 48px should the padding be 24px instead and margin -24px?

Reply:
Hi @mmalerba! I updated the changes based on your recommendation, but was wondering if I should change the padding to be 24px and margin -24px so the touch target is the accessible minimum 48px x 48px? Let me know your thoughts. Thanks for the consult!
[Latest commit](https://github.com/angular/components/pull/29351/commits/37d622067faeaf5f2e36cc142f9017d032f692e3)
[Updated screenshot](https://screenshot.googleplex.com/AiyY96F9SUYmacp)

Reply:
I think it should be the correct size if you add back the original styles you accidentally removed:
```
$offset: 2px;
top: 0 - $offset;
bottom: 0 - $offset;
left: 8px - $offset;
right: 8px - $offset;
border-radius: 50%;
```

Also, I notice that in some places we're setting `background: ...` for the `::after` element which clobbers the `background-clip` value. We should change those to `background-color` so they're not inadvertently changing the clip. We'll see what kind of issues we run into presubmitting. It's possible people are doing their own overrides of `background`, in which case I think we'd just have to apply `background-clip` with a higher specificity.

Reply:
Noted! Apologies for my confusion, returning the styles and updating the other `::after` styles as recommended to solve for background-clip.